codegenerator/cli/templates/static/codegen/src/globalState/GlobalState.res

```diff
@@ -907,7 +925,16 @@ let injectedTaskReducer = (
         chainFetchers,
       }
 
-      dispatchAction(SetRollbackState(inMemoryStore, chainManager))
+      chainFetcher.logger->Logging.childInfo({
+        "msg": "Finished rollback on reorg",
+        "entityChanges": {
+          "deleted": rollbackResult["deletedEntities"],
+          "upserted": rollbackResult["setEntities"],
+        },
```

Comment:
This might be a little misleading, this will just be the diff, maybe it should say finished diffing or something.